uberenv.py

```diff
@@ -870,24 +862,50 @@ def patch(self):
 
         # Update spack's config.yaml if clingo was requested
         if self.use_clingo:
-            concretizer_cmd = "{0} config --scope defaults add config:concretizer:clingo".format(self.spack_exe_path())
+            concretizer_cmd = "{0} config --scope defaults add config:concretizer:clingo".format(self.spack_exe(use_spack_env=False))
             res = sexe(concretizer_cmd, echo=True)
             if res != 0:
                 print("[ERROR: Failed to update spack configuration to use new concretizer]")
                 sys.exit(-1)
 
+    def create_spack_env(self):
+        # Create Spack Environment
+        print("[creating spack env]")
+        spack_create_cmd = "{0} env create -d {1} {2}".format(self.spack_exe(use_spack_env=False),
```

Comment:
The `-d` option should be inside the prefix. This ends up wherever you run uberenv from, which for me was inside Axom's source repo.

Reply:
spack env will now be created in prefix. docs also updated.